unix/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ansible = "<2.10"
```

Comment:
This is also a major win. If I'm following, this means that pipenv will pull the latest version of 2.9, meaning people get the most updated and secured version of ansible. And I think ansible follows semver closely, so there should be no breaking changes.